src/SpannerDriver.php

```diff
@@ -134,12 +174,19 @@ public function getInstance(string $instanceName): Instance
     /**
      * Returns a list of database names existing on a Spanner instance.
      *
+     * @param string $instanceName
+     *
      * @return array|string[]
+     * @throws GoogleException
      */
-    public function showDatabases(): array
+    public function listDatabases(string $instanceName = ''): array
```

Comment:
The default value for param that should be instances of some class should be `null` 
```suggestion
    public function listDatabases(string $instanceName = null): array
```